src/frontend/packages/lib_video/src/components/common/Player/createVideojsPlayer.ts

```diff
@@ -152,6 +160,9 @@ export const createVideojsPlayer = (
   });
 
   if (isMSESupported()) {
+    if (isP2pQueryEnabled && isP2PEnabled) {
```

Comment:
here `isP2PEnabled` is confusing. See my explanation about the name you use in the backend setting and in the front application.